core/store/orm/config.go

```diff
@@ -972,6 +972,12 @@ func (c Config) OperatorContractAddress() common.Address {
 	return *address
 }
 
+// OptimismGasFees enables asking the network for gas price before submitting
+// transactions, enabling compatibility with Optimism's L2 chain
```

Comment:
Might want to add a cautionary note against enabling this on any other network due to security concerns?

Reply:
What kind of security concerns are there?

Reply:
If we call estimate gas on mainnet this is a potential security concern. I do not recall why but @kaleofduty will be able to explain

Reply:
Seems like something that should go into docs, rather than the code.